apps/site/src/_pages/legal/terms/1_products.mdx

```diff
@@ -1,13 +1,19 @@
 ---
 title: Block Protocol Terms for Additional Products and Features
 shortTitle: Additional Product and Feature Terms
-location: /legal/terms/product
+location: /legal/terms/products
 redirect_from:
   - /legal/additional-product
+  - /legal/additional-products
   - /legal/product
+  - /legal/products
   - /legal/terms/additional-product
+  - /legal/terms/additional-products
+  - /legal/terms/products
```

Comment:
Do you want the original one too? `/legal/terms/product`

Reply:
```suggestion
  - /legal/terms/product
```